src/cases/components/roles-and-access/roles-and-access.component.ts

```diff
@@ -13,6 +13,8 @@ export class RolesAndAccessComponent implements OnInit, OnChanges {
   public exclusionsNotNamed: boolean = false;
   public legalOpsRoles: CaseRole[] = [];
   public namedLegalRoles: CaseRole[];
+  public namedAdminRoles: CaseRole[];
+  public adminRoles: CaseRole[];
```

Comment:
For consistency `adminRoles` and `namedAdminRoles` could have values set like with `legalOpsRoles` and `judicialRoles` properties e.g:

`public namedAdminRoles: CaseRole[] = [];`
`public adminRoles: CaseRole[] = [];`

Reply:
Sure